recipe/meta.yaml

```diff
@@ -62,12 +60,13 @@ test:
     - rio --help
     - rio info {{ RECIPE_DIR }}/test_data/test.tif  # [not win]
     - rio info {{ RECIPE_DIR }}\\test_data\\test.tif  # [win]
+    # 2022/4/25: Skip pytest because source files are large than 14Mb.
```

Comment:
... larger ...
why we want to skip tests just because this?  Are resource too limitted on builders?  If so, we should raise those limits.  Additionally we should not use '-k' option, if not really required.  Instead run pytest with '|| true' for unix and '| 1' on windows for preventing failure

Reply:
Cheng said that source_files can trigger security scanners. 
And test files (mainly images) are larger than a package itself. IMHO it's not good for end-users